ml_privacy_meter/utils/attack_utils.py

```diff
@@ -115,7 +115,7 @@ def get_per_class_indices(y, num_data_in_class, seed):
                                                 random_state=seed)
     """
     per_class_splitter = StratifiedShuffleSplit(n_splits=1,
-                                                train_size=0.5,
+                                                train_size=len(y)-num_classes,
```

Comment:
I thought this would be 1.0 or len(y). 

Separately, I only realized just now that we don't even need a split now, but we could keep this here for later splitting if we want.

Reply:
Sounds good